ConsoleGraphics/ConsoleProgressBar.cs

```diff
@@ -41,23 +54,34 @@ public class ConsoleProgressBar : IDisposable
         /// <param name="widthInCharacters">Size of progress bar in characters. Defaults to 40.</param>
         /// <param name="completedColor">Color for completed portion of progress bar. Defaults to Cyan.</param>
         /// <param name="remainingColor">Color for incomplete portion of progress bar. Defaults to Black.</param>
+        /// <param name="showETA">Show or not ETA</param>
         public ConsoleProgressBar(
             uint totalUnitsOfWork,
             int startingPosition = 0,
             uint widthInCharacters = 40,
             ConsoleColor completedColor = ConsoleColor.Cyan,
-            ConsoleColor remainingColor = ConsoleColor.Black)
+            ConsoleColor remainingColor = ConsoleColor.Black,
+            bool showETA = false)
```

Comment:
It looks like we're currently throwing away showETA passed into the constructor? Let's assign from this rather than setting to true when calling StartWork.

Reply:
This is then checked in the Draw method.
If you create the class supposing to display ETA then it will fire exception in case you dind't declare the Start time of your process!